lib/Parse/ParseStmt.cpp

```diff
@@ -21,6 +21,7 @@
 #include "clang/Sema/PrettyDeclStackTrace.h"
 #include "clang/Sema/Scope.h"
 #include "clang/Sema/TypoCorrection.h"
+#include "clang/Sema/SemaDiagnostic.h"
```

Comment:
Maybe it was related to previous work
I will check this. If possible, I will undo this